packages/server/tests/clients/servicesClient.ts

```diff
@@ -236,6 +236,24 @@ export default class ServicesClient {
         );
     };
 
+    async deployContract(contract, privateKey, gas = 100000) {
```

Comment:
This will not work for every `contract` provided. Let's refactor the function so that it uses the [EthereumFlow](https://github.com/hashgraph/hedera-sdk-js/blob/e696b77d3ee60b6004ef03723654a13aeb540812/src/EthereumFlow.js) we can then use the same function for deploying `basic` contract and `parentContract` and delete `createParentContract` all together

Reply:
Lets address this in the PR for erc20 contract deployment #231